src/gpodder/schema.py

```diff
@@ -278,6 +301,9 @@ def convert_gpodder2_db(old_db, new_db):
                 0,
                 row['sync_to_devices'],
                 None,
+                row['author'] or '',
```

Comment:
there's no way there could be a row['author'], row['categories'], row['keywords'] because they are new fields.
So I would prefer that you just put the `''`. If you used  row['author'] to remind us where we are in the long list of values, it makes sense but maybe a comment would be better

Reply:
resolved in [998c007](https://github.com/gpodder/gpodder/pull/1432/commits/998c007d8e12dcd69f238ab0d00942034c234fdc)